cookbooks/aws-parallelcluster-config/templates/default/init/fetch_and_run.erb

```diff
@@ -15,6 +15,7 @@ function error_exit () {
   script=`basename $0`
   echo "parallelcluster: ${script} - $1"
   logger -t parallelcluster "${script} - $1"
+  echo "${script} - $1" > /var/log/parallelcluster/headnode_bootstrap_error_msg
```

Comment:
Can we add here aslo ask users to check the corresponding logs for details? eg:"Additional info can be found in /var/log/.....log"?

Reply:
Yes we can.